README.md

```diff
@@ -133,4 +140,4 @@ WebHostingHub Glyphs
 - License: http://creativecommons.org/licenses/by/3.0/
 
 Splashy Icons (free as in free) 
-- Web page: http://splashyfish.com/icons/
```

Comment:
Should we revert the changes related to PR 2003 and close it? What do you suggest? Is the PR 2011 ok?